pkg/core/native/management.go

```diff
@@ -0,0 +1,72 @@
+package native
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/interop"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+)
+
+// Management is contract-managing native contract.
+type Management struct {
+	interop.ContractMD
```

Comment:
This contract has ID == 0, so we need to adjust one line in `initVerificationVM`: `if cs.ID < 0 {`. Although at the current moment Management contract doesnt'n have `verify` method.

Reply:
Fixed.